FutureFeatures.md

```diff
@@ -273,14 +273,19 @@ quadruple precision.
 
 ## Floating point library intrinsics
 
-These operations aren't needed because they can be implemented in WebAssembly
-code and linked into WebAssembly modules as at small size cost, and this avoids
-a non-trivial specification burden of their semantics/precision. Adding these
-intrinsics would allow for better high-level backend optimization of these
-intrinsics that require builtin knowledge of their semantics. On the other
-hand, a code generator may continue to statically link in its own
-implementation since this provides greater control over precision/performance
-tradeoffs.
+These operations aren't needed for the MVP because they can be implemented in
+WebAssembly code and linked into WebAssembly modules at small size cost (as is
+traditionally done through C libraries such as libm). This avoids a non-trivial
+specification burden for their semantics, precision, and performance.
```

Comment:
Additional advantages include:
 - developers (or tools/libraries/etc. made for developers) can make different tradeoffs of precision/robustness versus performance, while still getting deterministic results across implementations.
 - there's no temptation among implementations to "race to the bottom" in precision or robustness to compete for performance.
 - There are a *lot* of math functions, more than just the handful of trig and exponentiation variations listed here, and right now we have a very clear line of what belongs in the platform and what belongs in a library. Adding all these new functions would make it a lot less clear and open up the door to people wanting us to implement more functions in the future.